global/scripts/golang/test/run.sh

```diff
@@ -12,8 +12,15 @@ export INIT_SCRIPT="config.sh"
 
 touch coverage.xml
 
+test_folders=""
```

Comment:
They are not test folders. They are actually production code. Please give a more meaningful name to this variable. Or just keep it "folders".